### ISSUE TITLE
Entities Cannot Be Selected When Automation Creation

### ISSUE THREAD

**Issue by Betacart** [home-assistant/frontend#13578]
### The problem

When creating new automations for lights for example, I select set_mode for light. But there is no option to select entities, which I am setting mode for. When configure by YAML, it is OK, but entity is not visible, when switched back to UI. Without editing YAML, I am not able to select entity, for which I am setting light mode. When review old automations, entities also cannot be visible, but in YAML code, they are present.

### What version of Home Assistant Core has the issue?

2022.8.7

### What type of installation are you running?

Home Assistant OS

**Comment by spacegaier** [comment]
Can you please your automation YAML

**Comment by Betacart** [comment]
Hi,

no problem
`alias: DVEŘE - Dveře od 21:00
description: ""
trigger:
  - type: opened
    platform: device
    device_id: 22c43d545b4ac9c7d9831a1fcee28567
    entity_id: binary_sensor.predsin_vchod_opening
    domain: binary_sensor
condition:
  - condition: time
    after: "21:00:00"
    before: "22:00:00"
    weekday:
      - mon
      - tue
      - wed
      - thu
      - fri
      - sat
      - sun
action:
  - service: yeelight.set_mode
    data:
      mode: rgb
    target:
      entity_id:
        - light.obyvak_prouzek
        - light.obyvak_lampa
  - service: yeelight.set_color_scene
    data:
      brightness: 70
      rgb_color:
        - 255
        - 174
        - 0
    target:
      entity_id:
        - light.obyvak_prouzek
  - service: yeelight.set_color_scene
    data:
      brightness: 100
      rgb_color:
        - 255
        - 153
        - 0
    target:
      entity_id: light.obyvak_lampa
  - service: automation.turn_on
    data: {}
    target:
      entity_id:
        - automation.22_00
mode: single
`

<img width="1061" alt="Snímek obrazovky 2022-09-06 v 6 46 15" src="https://user-images.githubusercontent.com/6172834/188549286-16224ae3-fbd2-4039-a515-3147d048b4eb.png">


Jirka

**Comment by spacegaier** [comment]
I had a quick look at the service definition of `set_color_scene`. It requires the target to be of domain `light` and being created by the `yeelight` integration itself. Is that the case for `light.obyvak_prouzek`? Is that coming from the `yeelight` integration? If not, that would explain why the frontend is not offering that as a potential valid value.

And when you say "worked before"? Which HA version was that?

**Comment by Betacart** [comment]
Hi and thank you for analysis - yes, `light.obyvak.prouzek` is classic Yeelight Smart strip and belongs to `light` domain. And I already created that particular scene maybe 2 months back. When creating, I was able to select, which entities should be for that automation (there is also other lights I am powering on (`light.obyvak.lampa`)). I did not noticed, when exactly it stopped working, as I not touched this automation since then. I wanted to create new one and found this issue in latest HA version. I tried other service-based things and I am not ale to select any entity...

**Comment by spacegaier** [comment]
> I tried other service-based things and I am not ale to select any entity...

So it is a general issue for you with the entity picker and not specific to this automation and this entity? In that case, have you forced a clearing of your browser cache?
Does the entity picker e.g. in he HA logbook or history work for you?

**Comment by Betacart** [comment]
Hi,

I also not see it in the companion app - see screenshot. So, clearing cache does not help. I tried Chrome, Safari, Companion App - all with same result.
![IMG_7789](https://user-images.githubusercontent.com/6172834/188609780-72df5d05-99dd-48d1-8800-50c6b2092154.PNG)

**Comment by spacegaier** [comment]
See above:

> Does the entity picker e.g. in the HA logbook or history work for you?

**Comment by Betacart** [comment]
Yes, in the logbook, I can select entity with no problem (`light.obyvak_prouzek`).

**Comment by spacegaier** [comment]
Then I am afraid, I don't know what is going on here and I don't have any Yeelight products to try to reproduce myself.

**Comment by spacegaier** [comment]
Potentially a duplicate of https://github.com/home-assistant/frontend/issues/13566, if not integration specific.

**Comment by Betacart** [comment]
Hi, 

closing the issue, after installing 2022.9.0, entity can be selected again.
